lib/services/common/alarmManagement.js

```diff
@@ -41,7 +41,7 @@ function raise(alarmName, description) {
             description: description
         };
 
-        logger.error(context, 'Raising [%s]: %s', alarmName, description);
+        logger.error(context, 'Raising [%s]: %j', alarmName, description);
```

Comment:
What's the difference between `%s` and `%j`?

(Just curiosity... if some day I have known it, I have forgot :)

Reply:
(The doubt is in `%j`... `%s` I guess stand for string)